networkx/generators/duplication.py

```diff
@@ -138,7 +145,7 @@ def duplication_divergence_graph(n, p, seed=None):
         msg = "n must be greater than or equal to 2"
         raise nx.NetworkXError(msg)
 
-    G = nx.Graph()
+    G = (create_using or nx.Graph)()
```

Comment:
Same deal here - add checks to make sure DiGraph and MultiGraph are disallowed. Perhaps the easiest way to do this would be something like:

```python
G = nx.empty_graph(0, create_using=create_using)
if G.is_directed():
    raise
if G.is_multigraph():
    raise
```

Reply:
I've added checks for the duplication-divergence graphs. Did you have other graphs in `random_graphs.py` in mind?